LICENSE.md

```diff
@@ -2,21 +2,16 @@ The MIT License (MIT)
 
 Copyright (c) 2021 Haute École d'Informatique (HEI) de Madagascar
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
```

Comment:
Revert this file formatting to its previous state. Indeed your commit is only about removing the warnings related to Google check. The LICENSE file has nothing to do with that task. Remember to only do in your commit what your commit title suggests, and nothing more.